build.env

```diff
@@ -13,7 +13,7 @@ CSI_IMAGE_VERSION=canary
 
 # Ceph version to use
 BASE_IMAGE=docker.io/ceph/ceph:v16
-CEPH_VERSION=octopus
+BUILD_TAGS=ceph_preview
```

Comment:
can we document the ceph_preview thing ?

Reply:
>can we document the ceph_preview thing ?

```suggestion
# ceph_preview tag is required to access go-ceph APIs which are under preview.
BUILD_TAGS=ceph_preview
```

@humblec is this sufficient ?

Reply:
add a link to https://github.com/ceph/go-ceph/blob/master/docs/api-stability.md#preview here

Reply:
I do not think we should set `ceph_preview` as default. Make sure that APIs from go-ceph are stable and part of the latest release.

Reply:
`ceph_preview` is removed and go-ceph is upgraded to v0.13.0 in this pr instead